examples/add-API.graphql

```diff
@@ -0,0 +1,124 @@
+mutation {
```

Comment:
I got rid of those files because fixing formatting and deciding when to generate examples definitely takes some time and this is out of the scope of this issue. 
`saveQueryInExamples` is still in the code, but its body is commented out.